wagtail/admin/static_src/wagtailadmin/app/draftail.entry.js

```diff
@@ -10,29 +10,31 @@ import draftail, {
  * Entry point loaded when the Draftail editor is in use.
  */
 
+ // Expose Draftail package as a global.
+window.Draftail = Draftail;
```

Comment:
`Draftail` vs `draftail` is going to be rather confusing IMO.

Also it might accidentally expose some private APIs which people will start using, forcing us to maintain them and ensure backward compatibility. Maybe not such a strong argument since it's already the case in Python (we can't stop a user to import a private API but we still don't ensure backward compatibility).

Reply:
I agree it's a potential source of confusion but I couldn't find an alternative of similar simplicity I found good enough. For a more long-term solution I would like to do what I highlighted in https://github.com/wagtail/wagtail/issues/2315#issuecomment-360841058 but it would take "a couple of days" I haven't been able to find yet.

Reply:
For the potential private APIs / API breaking changes, [Draftail the package's public API](https://github.com/springload/draftail/blob/master/lib/index.test.js) is all documented – it might unintentionally expose some undocumented / private API, but I think this is unlikely. I tried to keep the public API as small as possible to limit this risk (only `DraftailEditor` has a big API surface, and it's already exposed outside this PR with the rich text features).